source/includes/code-examples/DeleteOneSync.cs

```diff
@@ -0,0 +1,70 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class DeleteOneSync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Delete a document using builders
+        WriteLine("Deleting a document with builders...");
+        var result = DeleteARestaurantBuilderSync();
+
+        WriteLine($"Deleted documents: {result.DeletedCount}");
+        // Extra space for console readability 
+        WriteLine();
+        
+    }
+
+    private static DeleteResult DeleteARestaurantBuilderSync()
+    {
+        // start-delete-one-builders-sync
+        var filter = Builders<Restaurant>.Filter
+            .Eq("cuisine", "Continental");
```

Comment:
Issue: Deleting a document by its cuisine does not ensure that we are deleting a specific document, since many documents have the cuisine "Continental". Since DeleteOne only deletes the first instance that matches the filter, there's no guarantee we know what's being deleted.

Suggestion: Change the filter to delete a specific known document, either by Name or RestaurantId or both of those.

Applies to Async method as well